frontend/model/contracts/group.js

```diff
@@ -229,14 +230,16 @@ DefineContract({
     'gi.contracts/group/invite': {
       validate: objectOf({
         inviteSecret: string, // NOTE: simulate the OP_KEY_* stuff for now
-        numInvites: number
+        numInvites: number,
+        invitee: string
       }),
       process (state, { data, meta }) {
         Vue.set(state.invites, data.inviteSecret, {
           generated: data.numInvites,
           creator: meta.username,
           responses: {},
-          status: 'valid'
+          status: 'valid',
+          invitee: data.invitee
```

Comment:
We already have `creator` generated from `meta.username`, so I think you can delete `invitee` and use `creator` instead

Reply:
@grg invitee is the person invited, not the creator.

Reply:
Ah, true. Hmm, I think let's wait for @sandrina-p's PR to be merged (#747), since it's almost done and also changes a lot of this stuff.

Reply:
Ok. once her PR is merged, I will come back and work on this part.